src/collar/oc_sys.lsl

```diff
@@ -720,14 +718,13 @@ default {
             g_iWaitRebuild = TRUE;
             PermsCheck();
             llSetTimerEvent(1.0);
-            llMessageLinked(LINK_ALL_OTHERS, LM_SETTING_REQUEST,"ALL","");
             if(llGetInventoryType(".settings") == INVENTORY_NOTECARD){
                 if(llGetInventoryKey(".settings") != g_kExistingSettings){
                     g_iSettingsReader=0;
                     g_kSettingsReader = llGetNotecardLine(".settings", g_iSettingsReader);
                     g_kExistingSettings = llGetInventoryKey(".settings");
                 }
-            }
+            } else llMessageLinked(LINK_ALL_OTHERS, LM_SETTING_REQUEST,"ALL","");
```

Comment:
Can you explain the reasoning for this change?

Reply:
1: if '.settings' card is found, it will be loaded first, and then all settings will be requested (line 754)
2: if '.settings' card is not found, then all settings will be requested (line 727)

Reply:
damn! I suppose I should have used 'LINK_SAVE' instead 'LINK_ALL_OTHERS'

Reply:
Yep clearly explained.